.github/workflows/build.yml

```diff
@@ -406,7 +406,7 @@ jobs:
       run: mvn test
 
   build-kotlin-macos:
-    name: Build Kotlin MacOS
+    name: Build Kotlin macOS
```

Comment:
brand -- there's a long story behind Apple's brand, but the form written here was never correct.